webapp/src/Twig/TwigExtension.php

```diff
@@ -208,6 +208,7 @@ public function getGlobals()
             'show_shadow_differences' => $this->tokenStorage->getToken() &&
                                          $this->authorizationChecker->isGranted('ROLE_ADMIN') &&
                                          $this->config->get('data_source') === DOMJudgeService::DATA_SOURCE_CONFIGURATION_AND_LIVE_EXTERNAL,
+            'doc_links' => $this->dj->getDocLinks(),
```

Comment:
Why is this a twig extension and not something the controller sets for the one template that uses it?

Reply:
Because the menu also uses it to decide if it should render an item, which is used on all team pages